homeassistant/components/ohme/__init__.py

```diff
@@ -0,0 +1,47 @@
+"""The ohme integration."""
+
+from dataclasses import dataclass
+import logging
+
+from ohme import OhmeApiClient
+
+from homeassistant import core
+from homeassistant.config_entries import ConfigEntry
+
+from .const import PLATFORMS
+from .coordinator import OhmeCoordinator
+
+_LOGGER = logging.getLogger(__name__)
+
+type OhmeConfigEntry = ConfigEntry[OhmeRuntimeData]
+
+
+@dataclass
+class OhmeRuntimeData:
+    """Store volatile data."""
+
+    client: OhmeApiClient
```

Comment:
tbh you can just make the client property in the coordinator "public" and then you don't need to store it again